python/docs/source/reference/pyspark.sql/functions.rst

```diff
@@ -279,6 +279,9 @@ Date and Timestamp Functions
     make_timestamp
     make_timestamp_ltz
     make_timestamp_ntz
+    try_make_timestamp
```

Comment:
shall we keep the function names sorted alphabetical?

Reply:
fixed